zebra/rib.h

```diff
@@ -169,7 +169,7 @@ struct route_entry {
 	/* Distance. */
 	uint8_t distance;
 
-	struct opaque *opaque;
+	struct re_opaque *re_opaque;
```

Comment:
I agree with renaming the struct, but renaming the field seems like a duplication to me. `re->opaque` looks simpler and cleaner than `re->re_opaque`.